src/utils/schemaValidators.js

```diff
@@ -1,10 +1,21 @@
-const OpenAPISchemaValidator = require('openapi-schema-validator').default;
-const openApi3Validator = new OpenAPISchemaValidator({
-    version: 3,
-    version3Extensions: {
-    }
-});
+
+const getOAI3Validator = (() => {
+    const OpenAPISchemaValidator = require('openapi-schema-validator').default;
+
+    let validator;
```

Comment:
Why do this as a function within function and not just have top-level variable `validator`?